core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -408,7 +408,8 @@ offerbook.warning.tradeLimitNotMatching=Your payment account has been created {0
   Your trade limit is: {1}\n\
   The min. trade amount of the offer is: {2}.\n\n\
   You cannot take that offer at the moment. Once your account is older than 2 months this restriction gets removed.
-
+offerbook.warning.offerWasAlreadyUsedInTrade=You cannot take that offer because you have taken it already earlier. \
+  It might be that the take offer attempt ended up in a failed trade.
```

Comment:
```suggestion
offerbook.warning.offerWasAlreadyUsedInTrade=You cannot take this offer because you already took it earlier. \
  It could be that your previous take-offer attempt resulted in a failed trade.
```